src/dom/emittermixin.js

```diff
@@ -51,17 +51,19 @@ const DomEmitterMixin = extend( {}, EmitterMixin, {
 	 *
 	 * @method module:utils/dom/emittermixin~EmitterMixin#listenTo
 	 */
-	listenTo( ...args ) {
-		const emitter = args[ 0 ];
-
+	listenTo( emitter, event, callback, options ) {
```

Comment:
If you used `listenTo( emitter, ...rest )` here, you'd be able to use `proxy.attach( ...rest );` and `EmitterMixin.listenTo.call( this, emitter, ...rest );` later in the code. IMO, if the method does not process these arguments and just passes them through, it's safer to use rest.